app/elm/View.elm

```diff
@@ -2,13 +2,23 @@ module View exposing (..)
 
 import Model exposing (..)
 import Html exposing (..)
+import Html.Attributes exposing (..)
 import Views.Navbar exposing (navbar)
+import Views.Location exposing (location)
+import Views.Events exposing (events)
 import Views.Search exposing (search)
 
 
 view : Model -> Html Msg
 view model =
-    div []
+    div [ class "flex" ]
         [ navbar model
-        , search model
+        , div [ class "w-100" ]
+            [ div [ id "myMap", class "w-100 h5" ] []
+            , div [ class "flex" ]
+                [ search model
+                  -- , location model
```

Comment:
I was thinking that `location` was a preliminary screen and then `map` and `search` would show up. I was not sure about how we are going to perform that navigation so I commented out for the moment.